user_tools/docs/index.md

```diff
@@ -0,0 +1,111 @@
+# spark-rapids-user-tools
+
+A python wrapper that provides a runtime environment that simplifies running cost and performance
+analysis using the RAPIDS Accelerator for Apache Spark across multiple cloud service providers.
+
+## Available commands
```

Comment:
Put qualification first, followed by bootstrap and then profiling, as that is the natural order of usage